### PR TITLE
feat: Alibaba Cloud Tablestore docs

### PR DESCRIPTION
**Pull request by DDDHuang**

### CODE REVIEW

File: en_US/rule/backend_tablestore.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+# 保存数据到 Alibaba Tablestore 数据库
```
Comment (mononym) [inline en_US/rule/backend_tablestore.md:1]: Title should be in English

File: en_US/rule/backend_tablestore.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+payload.k_float as k_float,
+clientid
+
+FROM
+ "#"
+```
+
+### Create Action
+
+`Table Name` and `ata Source` name, please fill in the `Table Name` and `Data Source` name as created (`Data Source` name can be empty string).
```
Comment (mononym) [inline en_US/rule/backend_tablestore.md:55]: first `Data Source` is missing the D

File: en_US/rule/backend_tablestore.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+![img](./assets/rule-engine/tablestore/create_action.png)
+
+Parameter List
+| Parameter | Definition |
+| --- | --- |
+| Measurement | Tablestore Measurement |
+| Data Source | Tablestore Data Source, default is empty string |
+| TimeStamp(microsecond) |In microsecondMQTT messages in EMQX are timestamped with millisecond precision by default and cannot be used directly. Fill in the blank value,the default value is the microsecond timestamp of the message arriving at EMQX |
```
Comment (mononym) [inline en_US/rule/backend_tablestore.md:64]: I think the beginning `In microsecond` text is not supposed to be there? Perhaps should be deleted?

Comment (mononym) [inline en_US/rule/backend_tablestore.md]: What does it mean when it says `cannot be used directly`?

Comment (mononym) [inline en_US/rule/backend_tablestore.md]: Does putting a value here mean the timestamp is overwritten for all messages?

File: en_US/rule/backend_tablestore.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+| TimeSeries Cache | Tablestore identifies whether the current data needs to be created or updated with timeline metadata, enabled by default |
```
Comment (mononym) [inline en_US/rule/backend_tablestore.md:65]: `metadata. Enabled by default`

**Review by mononym — CHANGES_REQUESTED** [review]

File: en_US/rule/backend_tablestore.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+Create a database instance (please refer to the official Tablestore documentation for the steps to create an instance).
+
+</br>Click `Instances` . `Time Series Table`. `Create Time Series Table`.
+
+![img](./assets/rule-engine/tablestore/instance_manage.png)
+
+
+Manage Time Series Table
+
+Click on the `Time Series Table` created in the previous step to enter the timeline management.
```
Comment (mononym) [inline en_US/rule/backend_tablestore.md]:
```suggestion
Click on the `Time Series Table` created in the previous step to enter the timeline management UI.
```

File: en_US/rule/backend_tablestore.md [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+Click `Data Management`, `Query Data`, and `Add Time Series` according to the business design.
+(This step can be skipped, Tablestore will automatically create a new `Time Series` when writing data that does not exist, the example does not operate on `Time Series`)
```
Comment (mononym) [inline en_US/rule/backend_tablestore.md]:
```suggestion
(This step can be skipped, Tablestore will automatically create a new `Time Series` when writing data that does not exist. The example does not operate on `Time Series`)
```

File: en_US/rule/backend_tablestore.md (hunk 2 above)
Comment (mononym) [inline en_US/rule/backend_tablestore.md]:
```suggestion
`Table Name` and `Data Source` name, please fill in the `Table Name` and `Data Source` name as created (`Data Source` name can be empty string).
```

File: en_US/rule/backend_tablestore.md (hunk 3 above)
Comment (mononym) [inline en_US/rule/backend_tablestore.md]:
```suggestion
| TimeStamp(microsecond) | MQTT messages in EMQX are timestamped with millisecond precision by default and cannot be used directly. Fill in the blank value, the default value is the microsecond timestamp of the message arriving at EMQX |
```

File: en_US/rule/backend_tablestore.md (hunk 4 above)
Comment (mononym) [inline en_US/rule/backend_tablestore.md]:
```suggestion
| TimeSeries Cache | Tablestore identifies whether the current data needs to be created or updated with timeline metadata. Enabled by default |
```

**Review by mononym — CHANGES_REQUESTED** [review]

File: en_US/rule/backend_tablestore.md [hunk 7]
```diff
@@ -0,0 +1,95 @@
+# 保存数据到 Alibaba Tablestore 数据库
+
+Create a database instance (please refer to the official Tablestore documentation for the steps to create an instance).
+
+</br>Click `Instances` . `Time Series Table`. `Create Time Series Table`.
+
+![img](./assets/rule-engine/tablestore/instance_manage.png)
+
+
+Manage Time Series Table
```
Comment (gsychev) [inline en_US/rule/backend_tablestore.md]:
```suggestion
## Manage Time Series Table
```

File: en_US/rule/backend_tablestore.md [hunk 8]
```diff
@@ -0,0 +1,95 @@
+# 保存数据到 Alibaba Tablestore 数据库
+
+Create a database instance (please refer to the official Tablestore documentation for the steps to create an instance).
```
Comment (gsychev) [inline en_US/rule/backend_tablestore.md]:
```suggestion
## Create a database instance

> Please refer to the official Tablestore documentation for the steps to create an instance.
```

File: en_US/rule/backend_tablestore.md [hunk 9]
```diff
@@ -0,0 +1,95 @@
...
+
+Tablestore supported data types:
+
+- int
+- float
+- boolean
+- string
+- binary
+
+These formats can be automatically recognized and classified by the rules engine. String-type data, however, is handled as binary data by default to ensure best compatibility. If you need to specify a field value as a string, you can fill the field into the `String Fields` when you create the action. After that, the engine will handle the field value in the string format.
```
Comment (gsychev) [inline en_US/rule/backend_tablestore.md]:
```suggestion
These types can be automatically recognized and classified by the rules engine. String data, however, is handled as a binary by default to ensure the best compatibility. If you need to specify a field value as a string, you can fill the field into the `String Fields` when you create the action. After that, the engine will handle the field value as a string.
```

File: en_US/rule/backend_tablestore.md (hunk 3 above)
Comment (DDDHuang) [reply en_US/rule/backend_tablestore.md]: Use 
```SQL
SELECT * FROM "#"
```
Rule engine will `SELECT` message's timestamp as a key-value, and it is in millisecond. 
But tablestore need microsecond timestamp.

Comment (DDDHuang) [reply en_US/rule/backend_tablestore.md]: Only the timestamp written to the database will be overwritten.

**Review by terry-xiaoyu — APPROVED** [review]

**Pull request merged**